README.md

```diff
@@ -295,7 +295,7 @@ The relayer is configured via a JSON file, the path to which is passed in via th
 
```

Comment:
It's a bit odd that there's no description for `decider-host`. Even a token description would be good. The description of the semantics when this field is omitted should be included here.

Reply:
I got rid of `decider-host` altogether. See https://github.com/ava-labs/awm-relayer/pull/344#discussion_r1690526033